app/src/screens/Home/index.tsx

```diff
@@ -51,17 +53,30 @@ const styles = StyleSheet.create({
 const Home: React.FC = () => {
   const dispatch = useDispatch<Dispatch>();
   const user = useSelector((state: State) => state.user);
+  const navigation = useNavigation<StackNavigationProp<'App'>>();
 
   const handleLogout = async () => {
+    navigation.reset({ routes: [{ name: 'Busy' }] });
+
     const logoutAction = await dispatch(logout());
     if (logout.rejected.match(logoutAction)) {
       console.error(logoutAction.payload || logoutAction.error);
+      navigation.reset({
```

Comment:
Do we really want to replace "the whole navigation state" for unsuccessful logout? Maybe the user might want to go back to previous page?

Reply:
It seems like when we have an "unsuccessful logout" the user state is undefined behavior, so I thought it was better to just drop them at the login screen to reinitialize the whole user state.